gradle.properties

```diff
@@ -49,7 +49,7 @@ typesafeConfigVersion=1.3.1
 gradleGitPluginVersion=2.2.0
 
 # Version and base tags can be overridden at build time
-pravegaVersion=0.4.0-SNAPSHOT
+pravegaVersion=0.3.1-SNAPSHOT
```

Comment:
This doesn't look right. Why are we going back?

Reply:
It looks like an accidental change, this is clearly incorrect.